client/src/components/WriteToClipboard/WriteToClipboard.js

```diff
@@ -83,10 +83,7 @@ export class WriteToClipboard extends React.Component {
             </Fragment>
           }
           subtitle={modal_active && !copy_success && copy_status_message}
-          body={
-            modal_active &&
-            !copy_success && <div tabIndex="0">{text_to_copy}</div>
-          }
+          body={modal_active && copy_success && text_to_copy}
```

Comment:
Dropped a `!` on the `copy_success`. 

Also, I think this tab-able content was intentional because the purpose of this is to let a user manually copy some text when the clipboard access fails, I think a non-screen reader keyboard navigator needs to be able to focus on the text to easily copy just it?

I could be wrong on that, probably worth a little research.